pkg/builder/builder.go

```diff
@@ -82,6 +82,10 @@ const (
 	// Scrutinize constants
 	scrutinizeMountName       = "scrutinize"
 	scrutinizeTermGracePeriod = 0
+	// Environment variables to allow vcluster scrutinize to get the password
+	// from a secret
+	superuserPasswordSecretNamespaceEnv = "SUPERUSER_PASSWD_NAMESPACE"
```

Comment:
Does scrutinize already read this env var? I can't seem to find it.

Reply:
No that will part of a separate PR to add the logic to read this. But scrutinize already reads NMA certs from secret so I will reuse the logic.

Reply:
Got it. Suggest we rename this to be: PASSWORD_SECRET_NAMESPACE and PASSWORD_SECRET_NAME